packages/charts/react-charts/library/src/components/VerticalBarChart/VerticalBarChart.tsx

```diff
@@ -821,11 +814,29 @@ export const VerticalBarChart: React.FunctionComponent<VerticalBarChartProps> =
         enabledWrapLines={props.enabledLegendsWrapLines}
         overflowText={props.legendsOverflowText}
         {...props.legendProps}
+        selectedLegends={selectedLegends}
+        onChange={_onLegendSelectionChange}
```

Comment:
is selectedLegends already implemented in v9 legends yet.

Reply:
I was talking about this commit. https://github.com/microsoft/fluentui/commit/608336dee79ebb116cc5ed3d0775fc7385a028c8

Anush can enable it next.